File: app/src/main/java/com/cappielloantonio/play/subsonic/models/ArtistWithAlbumsID3.java
package com.cappielloantonio.play.subsonic.models;

import com.tickaroo.tikxml.annotation.Element;
import com.tickaroo.tikxml.annotation.Xml;

import java.util.ArrayList;
import java.util.List;

@Xml
public class ArtistWithAlbumsID3 extends ArtistID3 {
    @Element(name = "album")
    protected List<AlbumID3> albums;

    public List<AlbumID3> getAlbums() {
        if (albums == null) {
            albums = new ArrayList<>();
        }
        return this.albums;
    }

    public void setAlbums(List<AlbumID3> albums) {
        this.albums = albums;
    }
}

File: app/src/main/java/com/cappielloantonio/play/subsonic/models/ArtistWithAlbumsID3.kt
package com.cappielloantonio.play.subsonic.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
class ArtistWithAlbumsID3 : ArtistID3(), Parcelable {
    @SerializedName("album")
    var albums: List<AlbumID3>? = null
}